platform/administer/templates/parameters.mdx

```diff
@@ -0,0 +1,422 @@
+---
+title: Parameters
+sidebar_label: Parameters
+sidebar_position: 3
+---
+import Flow, { Step } from "@site/src/components/Flow";
+
+# Parameters
+
+Template parameters enable customization of virtual cluster templates. Platform administrators define a set of parameters that expose specific fields to users during virtual cluster creation. These parameters allow customization of values like replica counts, image tags, or ingress domains without granting access to the full configuration.
+
+When a user creates a virtual cluster from a parameterized template, the platform presents a form that collects values for each parameter. These values are injected into the rendered configuration using Go templating syntax (`{{ .Values.parameterName }}`).
+
+
+This provides some flexibility within defined boundaries. Developers can adjust some behaviors—such as increasing memory requests or enabling optional apps—while platform teams can still retain control over foundational settings like network configuration, cluster version, and permissions.
+
+Each parameter supports type definitions, validation rules, default values, and allowed value lists. These controls ensure that inputs are valid and align with standards.
+
+Parameters work well for supporting self-service provisioning at scale and also maintain governance, consistency, and operational safety across environments.
+
+
+## Parameter types and validation
+
+When defining parameters in a vCluster template, you must specify the expected input type and optionally enforce validation rules.
+
+### Supported parameter types
+
+Templates support basic parameter types for collecting user input during cluster creation:
+
+| Type          | Use case                                               |
+|---------------|--------------------------------------------------------|
+| [`String`](#string-parameter)    | Names, labels, image tags, configuration values        |
+| [`Number`](#string-parameter)    | Replica counts, port numbers, resource limits          |
+| [`Boolean`](#boolean-parameter)  | Feature toggles, enable/disable flags                  |
+| [`Password`](#password-parameter)| API tokens, secrets, sensitive input values            |
+| [`Multiline`](#multiline-parameter)| Configuration files, startup scripts, long descriptions |
```

Comment:
Unsure if its a docs convention to uppercase these, but it might be better to lowercase them. That way it matches the casing in [the examples](https://deploy-preview-840--vcluster-docs-site.netlify.app/docs/platform/next/administer/templates/parameters/#number-parameter) and they are not tempted to capitalize it in config. I'm not sure if the backend will accept the following, for example:
```yaml
- variable: replica_count
  type: Number # uppercase instead of lowercase
```